groups/bace-iot-platform/measurements-webhook/payloads/testPayload.json

```diff
@@ -0,0 +1,118 @@
+[
```

Comment:
Can you reformat this file with Visual Studio Code (or something) to fix indentation and make sure there is a newline at the end.